tests/gem5/cpu_tests/ref/FloatMM

```diff
@@ -1,6 +0,0 @@
-gem5 Simulator System.  http://gem5.org
```

Comment:
I don't think you meant to delete this file :D

Reply:
Yeah I noticed that and changed it back locally, I'll fix it on my next push!